include/mbedtls/ssl.h

```diff
@@ -405,6 +405,18 @@
 #define MBEDTLS_SSL_CID_TLS1_3_PADDING_GRANULARITY 16
 #endif
 
+#if !defined(MBEDTLS_SSL_TLS1_3_TICKET_AGE_TOLERANCE)
+#define MBEDTLS_SSL_TLS1_3_TICKET_AGE_TOLERANCE 6000
+#endif
+
+#if !defined(MBEDTLS_SSL_TLS1_3_TICKET_NONCE_LENGTH)
+#define MBEDTLS_SSL_TLS1_3_TICKET_NONCE_LENGTH 32
```

Comment:
The disadvantage of doing it this way is to change the default one now needs to make identical changes in two places

Reply:
Indeed. But in practice, I don't recall us changing the default for any of thse options in the past 6½ years.

Anyway, if we changed the organization, this pull request wouldn't be the place to do it. It would require some thought about the new design and backward compatibility.